docs/contributing/documentation.mdx

```diff
@@ -54,9 +54,9 @@ Webiny docs repository uses [docusaurus](https://v2.docusaurus.io/) as the syste
 To run the docs portal locally on your machine:
 
 1. `git clone https://github.com/webiny/docs.webiny.com.git`
-2. `cd docs.webiny.com/website`
-3. `npm install`
-4. `npm start`
+2. `cd docs.webiny.com`
+3. `yarn`
+4. `yarn start`
 
 All the articles are inside the `/docs` folder. Edit `/website/sidebars.js` if you need to manage the items in the sidebar.
```

Comment:
`/website/sidebars.js` should just be `sidebars.js` as well.

Reply:
Ah, that's right, Pavel :). I missed that. 
I updated '/docs' folder to 'docs' folder. I hope it is okay?

Reply:
We don't have hard rule on that, the `/` just makes it  very clear that the `docs` folder is located in the root of the repository.